### PR TITLE
Optional MOTO-Flagging for Reseller Scheme-TXs

### PR DESCRIPTION
**Pull request by pc-coholic**

### CODE REVIEW

File: src/pretix/plugins/stripe/payment.py [hunk 1]
```diff
@@ -97,6 +98,30 @@ def settings_content_render(self, request):
 
     @property
     def settings_form_fields(self):
+        if 'pretix_resellers' in [p.module for p in get_all_plugins()]:
+            moto_settings = [
+                ('reseller_moto',
+                 forms.BooleanField(
+                     label=_('Enable MOTO payments for Resellers'),
+                     help_text=(
+                         _('Gated feature, that needs to be enabled for your account by Stripe support first.') +
```
Comment (raphaelm) [inline src/pretix/plugins/stripe/payment.py]:
```suggestion
                         _('Gated feature (needs to be enabled for your account by Stripe support first)') +
```
?

File: src/pretix/plugins/stripe/payment.py [hunk 2]
```diff
@@ -97,6 +98,30 @@ def settings_content_render(self, request):
 
     @property
     def settings_form_fields(self):
+        if 'pretix_resellers' in [p.module for p in get_all_plugins()]:
+            moto_settings = [
+                ('reseller_moto',
+                 forms.BooleanField(
+                     label=_('Enable MOTO payments for Resellers'),
```
Comment (raphaelm) [inline src/pretix/plugins/stripe/payment.py]:
```suggestion
                     label=_('Enable MOTO payments for resellers'),
```

File: src/pretix/plugins/stripe/payment.py [hunk 3]
```diff
@@ -643,6 +683,11 @@ def _handle_payment_intent(self, request, payment, intent=None):
                     amount=self._get_amount(payment),
                     currency=self.event.currency.lower(),
                     payment_method=request.session['payment_stripe_payment_method_id'],
+                    payment_method_options={
+                        'card': {
+                            'moto': self.is_moto(request, payment),
```
Comment (raphaelm) [inline src/pretix/plugins/stripe/payment.py]: Is it safe to send moto=False on a Stripe account that is not approved for MOTO?

Comment (pc-coholic) [reply src/pretix/plugins/stripe/payment.py]: "Safe" as in "works" or "won't get you flagged for SAQ D"?

Comment (raphaelm) [reply src/pretix/plugins/stripe/payment.py]: Ideally both ;)

Comment (raphaelm) [reply src/pretix/plugins/stripe/payment.py]: Was thinking of "works", though

**Comment by raphaelm** [comment]
@pc-coholic Sorry, built some conflicts there :( Can you check them?

**Pull request merged**